test/integration/mount_start_test.go

```diff
@@ -81,7 +81,7 @@ func TestMountStart(t *testing.T) {
 func validateStartWithMount(ctx context.Context, t *testing.T, profile string) {
 	defer PostMortemLogs(t, profile)
 
-	args := []string{"start", "-p", profile, "--memory=2048", "--mount", "--mount-gid", mountGID, "--mount-msize", mountMSize, "--mount-port", mountPort, "--mount-uid", mountUID}
+	args := []string{"start", "-p", profile, "--memory=2048", "--mount", "--mount-gid", mountGID, "--mount-msize", mountMSize, "--mount-port", mountPort, "--mount-uid", mountUID, "--no-kubernetes"}
```

Comment:
are we changing this test to only test with no-kubernetes ? if yes why

Reply:
I was going to change it to only test with `--no-kubernetes` since we're just checking that the `start --mount` flags are being respected so having k8s running is not important, this will also speed up the test since we're starting two minikube instances and they'll start much faster with `--no-kubernetes`.